web-ui/src/main/resources/catalog/views/geocat/config.js

```diff
@@ -43,52 +43,113 @@
         })
       };
 
-      var resolutions = [
-        4000, 3750, 3500, 3250, 3000, 2750, 2500, 2250, 2000, 1750, 1500, 1250,
-        1000, 750, 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1.5, 1, 0.5
-      ];
-
-      var matrixIds = [];
-      for (var i = 0; i < resolutions.length; i++) {
-        matrixIds.push(i);
+      if(gnGlobalSettings.srs == "EPSG:4326" ) {
+
+         // Configuration for 4326
+
+         var projection = ol.proj.get('EPSG:3857');
```

Comment:
This is projection of base layer used by https://server.arcgisonline.com/arcgis/rest/services/World_Topo....